index.html

```diff
@@ -0,0 +1,168 @@
+      <button class="x-mark">&times;</button>
+      <ul id="menu-list">
+        <li><a id="" href="index.html">Home</a></li>
+        <li><a id="" href="about.html">About</a></li>
+        <li><a id="" href="#">Contact</a></li>
+      </ul>
+    </section>
+    <!-- Home page Intorduction -->
+    <main class="bg-gray">
+      <section class="introduction home-intro bg-gray font">
+        <h1>"Unleash your curiosity at the library."</h1>
+        <h2>ABREHOT Library</h2>
+        <div class="p-2 border-1 bg-gray">
+          <p class="p-1">
+            "Embark on a journey of discovery and endless learning as you
+            immerse yourself in the boundless treasures of knowledge and
+            imagination available at your fingertips in our library."
+          </p>
+        </div>
+        <p class="tbm-header bg-gray">25<sup>th</sup> May 2023</p>
+        <p class="nyerere">@Arat Killo, Addis Ababa</p>
+      </section>
+      <!-- Home page Main Part -->
+      <section class="main-part font">
+        <h2>OUR SERVICES</h2>
+        <div class="dash">_____</div>
+        <div class="main-div">
+          <div>
+            <img src="images/book_borrowing.png" alt="book Logo" />
+            <h3>Borrow</h3>
+            <p>Books are a uniquely portable magic.Enjoy by being Member.</p>
+          </div>
+          <div>
+            <img src="images/audio-book.png" alt="Audiobook Logo" />
+            <h3>Audio Book</h3>
+            <p>
+              An audiobook is a great way to escape into a story while you're
+              doing other things.
+            </p>
+          </div>
+          <div>
+            <img src="images/place.jpg" alt="place" />
+            <h3>Corner</h3>
+            <p>
+              The best place to study, quiet, well-lit, comfortable, and free
+              from distractions.
+            </p>
+          </div>
+          <div>
+            <img class="bg-red" src="images/coffee.jpg" alt="Coffee" />
+            <h3>Coffee</h3>
+            <p>
+              When you drink coffee ideas march in like the army.Enjoy our
+              Coffee.
+            </p>
+          </div>
+          <div>
+            <img src="images/internet.jpg" alt="Internet" />
+            <h3>Internet</h3>
+            <p>
+              Enjoy our High Bandwidth Internet.we support wifi and Ethernet.
+            </p>
+          </div>
+        </div>
+        <div class="last-main-div">
+          <h3 class="bg-red h3-one">LIBRARIES FUEL IMAGINATION</h3>
+        </div>
+        <h3>SEE OUR LIBRARY RESOURCE</h3>
+      </section>
+      <!-- Featured Partcipants -->
+      <section class="investors font">
+        <h2>Featured Advisors</h2>
+        <div class="dash">_____</div>
+        <div class="partcipants"></div>
+        <button class="none">
+          MORE <i class="fa-solid fa-greater-than fa-rotate-90"></i>
+        </button>
+      </section>
+      <section class="none-mobile last-section">
```

Comment:
- [ ] Good job by adding a partner section as required by the previous reviewer but I think you can improve your design because it looks different from the original design. Please make sure to stick to the original design to give a good user experience and make your website great 👍 

#### YOUR DESIGN
![part](https://user-images.githubusercontent.com/6615394/235008744-796248d3-937e-40b2-b17b-ced95c90c2d8.PNG)



#### template
![234959634-e2a93301-ec9f-4521-ac2e-fbb3562620bc](https://user-images.githubusercontent.com/6615394/235008465-6f24398a-0a09-48cc-8116-df503792e87c.png)